php-fpm/templates/default/php-fpm.conf.erb

```diff
@@ -1,5 +1,5 @@
 [global]
-pid = <%= node["php-fpm"]["prefix"] %>/var/run/php-fpm.pid
+pid = <%= node["php-fpm"]["pid"] %>
```

Comment:
This is a non-breaking changes for our own `php5-easybib` packages, but makes this configuration work with the ondreij PPA.